locales/fr-fr/case-studies.json

```
@@ -596,6 +596,16 @@
         {
           "text": "Blog",
           "to": "https://protectai.com/blog/how-protect-ai-is-shaping-the-future-of-llm-security-with-ebpf"
+          }
+       ]
+    },
+    {
+          "title": "Upwind Security",
+    "description": "<strong> Upwind Security </strong> utilise eBPF pour fournir une sécurité cloud approfondie en temps réel/à l'exécution à grande échelle.",
+    "links": [
+      {
+        "text": "Blog",
+        "to": "https://www.upwind.io/glossary/what-is-ebpf-security#toc-section-0"
```

Please preserve the formatting. Two spaces per indent level, this should give something like:

```suggestion
         }
       ]
     },
     {
       "title": "Upwind Security",
       "description": "<strong> Upwind Security </strong> utilise eBPF pour fournir une sécurité cloud approfondie en temps réel/à l'exécution à grande échelle.",
       "links": [
         {
           "text": "Blog",
           "to": "https://www.upwind.io/glossary/what-is-ebpf-security#toc-section-0"
```

Here and in the other files.

Should be easily doable by copy-pasted from well-formated JSON items, or by using your IDE to lint the JSON file.